Source/AssistantV2/Models/MessageInputOptions.swift

```diff
@@ -81,16 +90,18 @@ public struct MessageInputOptions: Codable, Equatable {
      - returns: An initialized `MessageInputOptions`.
      */
     public init(
-        debug: Bool? = nil,
         restart: Bool? = nil,
         alternateIntents: Bool? = nil,
+        spelling: MessageInputOptionsSpelling? = nil,
+        debug: Bool? = nil,
```

Comment:
This is a breaking change (changing the position of an existing parameter).  Is that intentional?

Reply:
@mamoonraja we need to check if this is an issue in all sdks. its fine in .net

Reply:
@mkistler I've updated the parameter position.